pandas/core/ops.py

```diff
@@ -731,7 +700,9 @@ def safe_na_op(lvalues, rvalues):
                                                   lambda x: op(x, rvalues))
             raise
 
-    def wrapper(left, right, name=name, na_op=na_op):
+    def wrapper(left, right, na_op=na_op):
+        # TODO: `na_op` does not belong in Series.__{op}__ signature.  But this
+        # is needed here for closure/namespace purposes ATM.
```

Comment:
can you add doc-strings. this is extremely useful to future readers.

Reply:
Any docstring added here would become the docstring for `pd.Series.__foo__`, which I doubt is what you have in mind here.

Reply:
right, so just a comment then which describes the input & types